src/components/Jobs/JobSelector/JobSelector.utils.ts

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { ServerErrors } from '../../../modals/commonTypes'
+import { showError } from '../../common'
+import { getJobs } from '../Service'
+
+export const appSelectorStyle = {
```

Comment:
Please remove this and reuse the app selector utils style

Reply:
Yes. Removed this & now using the AppSelector.